src/main/java/org/gridsuite/modification/server/impacts/SimpleElementImpact.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.impacts;
+
+import com.powsybl.iidm.network.IdentifiableType;
+import lombok.*;
+
+import java.util.Set;
+
+/**
+ * @author Slimane Amar <slimane.amar at rte-france.com>
+ */
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Setter
+@Getter
+@Builder
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+@ToString
+public class SimpleElementImpact extends AbstractElementImpact {
+    public enum SimpleImpactType {
+        CREATION,
+        MODIFICATION,
+        DELETION
+    }
+
+    private SimpleImpactType impactType;
+
+    private String elementId;
```

Comment:
Why elementId and not equipmentId, what's the difference ? Same for elementType

Reply:
Currently in discussion with @geofjamg